api/HTMLInputElement.json

```diff
@@ -1355,6 +1402,53 @@
           }
         }
       },
+      "useMap": {
```

Comment:
Can you add "usemap" to input.json? It's been supported since the dawn of time, using the same data as for this entry would work.